pages/common/if.md

```diff
@@ -1,27 +1,28 @@
 # if
 
 > Simple shell conditional.
+> See also: `test`, `[`.
 
-- Echo a different thing depending on a command's success:
+- Executed two different commands based on a condition:
 
-`if {{command}}; then echo "success"; else echo "failure"; fi`
+`if {{command}}; then {{echo "true"}}; else {{echo "false"}}; fi`
 
-- Full if syntax:
+- Check if a variable is defined:
 
-`if {{condition}}; then echo "true"; else echo "false"; fi`
+`if [[ -n {{$VARIABLE}} ]]; then {{echo "defined"}}; else {{echo "not defined"}}; fi`
```

Comment:
A variable can be defined or not, and have an empty value or not. These are two different things for a shell. This works in zsh:
```
% unset FOO
% if (( ${+FOO} )); then echo "FOO is defined"; else echo "FOO is not defined"; fi
FOO is not defined
% FOO=
% if (( ${+FOO} )); then echo "FOO is defined"; else echo "FOO is not defined"; fi
FOO is defined
% if [[ -n ${FOO} ]]; then echo "FOO is nonempty"; else echo "FOO is empty"; fi
FOO is empty
% FOO=1
% if [[ -n ${FOO} ]]; then echo "FOO is nonempty"; else echo "FOO is empty"; fi
FOO is nonempty
```
I know it's tricky...  :- )